contracts/LifCrowdsale.sol

```diff
@@ -13,13 +13,13 @@ contract LifCrowdsale is Ownable, Pausable {
   LifToken public token;
 
   // start and end of the public presale
-  uint256 public publicPresaleStartBlock;
-  uint256 public publicPresaleEndBlock;
+  uint256 public publicPresaleStartTime;
```

Comment:
I think `Time` as suffix is misleading here. It sounds like the time of the day, but of course it's a timestamp, meaning the date and the time. Timestamp as suffix would be too long so what if we use `At`? `publicPresaleStartAt`, `publicPresaleEndAt`, `startAt`, `end1At`, `end2At` etc.

What do you think?

Reply:
if we are consistent about that, why not..

Reply:
I prefer using `time` on the name instead of `at`. because it can be at a block number too. For me time its ok, there is two ways to verify "time" conditions on solidity, blocks or timestamps, and I think using time is enough and doesnt bring any misleading.

Reply:
And we are already using `time` on VestedPayment contract.

Reply:
I know we are using time in vested payment, I suggested to change it to timestamp, you preferred to keep it and I said "ok" because it was just one variable. Now it's a more widespread thing. Think why it's `block.timestamp` instead of `block.time`. We should at least be consistent with that, again I think time is more about time of the day, not an exact moment in time